### DISCUSSION TITLE
Add a facilitator for editing along the Y-axis

### DISCUSSION THREAD

**Discussion by Djard** [audacity/audacity#4744] [Far future]
When I use, say, the Time-stretch tool or Amplify tool, I zoom in extensively; so that I can place the start and end points of the edit exactly on the Y-axis. Doing so prevents creation of pop/click artifact. Now the first zooming in to the starting point is easy because Audacity keep the mark centered on the screen. But scrolling to the end of the section is a labor intensive task. Adding a feature that automatically place the cursor on the Y-axis, or being able to zoom in to a second centering view would be a valuable asset in the already wonderful Audacity.

**Comment by Paul-Licameli** [comment]
Maybe you can use “Fine Zero Crossings” (the Z key) for fine adjustments of the selection to avoid clicks.

**Reply by Djard** [reply]
Yay! Thanks for the tip of using the hotkey Z to snap the cursor to the X-axis: it helps me tremendously.
